README.md

```diff
@@ -13,12 +13,23 @@ Once you download the installation package, you can rename *WinSW.exe* to any na
 
 See the [project manifest](MANIFEST.md).
 
+## Supported platforms
+
+WinSW offers executables for .NET Framework 2.0, 4.0 and 4.6.1.
+It can run on Windows platforms which have these versions of .NET framework installed.
+For systems without .NET Framework, the project provides native 64bit and 32bit executables which are based on .NET Core 3.1.
+
+More executables can be added upon request.
+
 ## Download
 
-Starting from WinSW v2, the releases are being hosted on [GitHub](https://github.com/winsw/winsw/releases) and [NuGet](https://www.nuget.org/packages/WinSW/).
+WinSW binaries are available on [GitHub Releases](https://github.com/winsw/winsw/releases) and [NuGet](https://www.nuget.org/packages/WinSW/).
 
-Due to historical reasons, the project also uses the [Jenkins](https://jenkins.io/) Maven repository as a secondary source. 
-Binaries are available [here](https://repo.jenkins-ci.org/releases/com/sun/winsw/winsw/). 
+Alternative sources:
+
+* [Maven packaging](https://github.com/jenkinsci/winsw-maven-packaging) for executables, hosted by the [Jenkins project](https://jenkins.io/). 
+Binaries are available [here](https://repo.jenkins-ci.org/releases/com/sun/winsw/winsw/).
+* [Puppet](./doc/puppetWinSW.md), currently outdated (WinSW 1.x). Binaries are available on [Puppet Forge](https://forge.puppet.com/kenmaglio/winsw)
```

Comment:
FTR I created https://github.com/kenmaglio/winsw/issues/9

Reply:
It's probably better to distribute the binaries on Chocolatey.

Reply:
Created https://github.com/winsw/winsw/issues/503 for the backlog